### PR TITLE
fix: get benchmarks to compile

### PR DESCRIPTION
**Pull request by gregdhill**
Signed-off-by: Gregory Hill <gregorydhill@outlook.com>

### CODE REVIEW

File: crates/relay/src/benchmarking.rs [hunk 1]
```diff
@@ -24,6 +24,9 @@ use vault_registry::{
     Pallet as VaultRegistry,
 };
 
+#[cfg(test)]
+use crate::Pallet as Relay;
```
Comment (nud3l) [inline crates/relay/src/benchmarking.rs:28]: The relay benchmark for `store_block_header` will be quite a t the low end if we assume that no forks/re-orgs are happening. Likely better to include a benchmark that causes a re-org to be on the conservative side. What do you think?

We can also do that later due to timining now.

File: parachain/runtime/src/lib.rs [hunk 2]
```diff
@@ -1048,7 +1048,7 @@ impl_runtime_apis! {
             add_benchmark!(params, batches, relay, Relay);
             add_benchmark!(params, batches, vault_registry, VaultRegistry);
             add_benchmark!(params, batches, fee, Fee);
-            add_benchmark!(params, batches, nomination, Nomination);
+            // add_benchmark!(params, batches, nomination, Nomination);
```
Comment (sander2) [inline parachain/runtime/src/lib.rs:1051]: Probably should undo this?

File: crates/issue/src/benchmarking.rs [hunk 3]
```diff
@@ -68,7 +71,7 @@ fn mine_blocks<T: crate::Config>() {
         .build();
 
     let mut prev_hash = block.header.hash;
-    for _ in 0..100 {
+    for _ in 0..600 {
```
Comment (sander2) [inline crates/issue/src/benchmarking.rs:74]: any particular reason for this? If so, would be good to add a comment to explain the reason

File: parachain/runtime/src/lib.rs (hunk 2 above)
Comment (gregdhill) [reply parachain/runtime/src/lib.rs:1051]: There are no nomination benchmarks at the moment, we should add in a separate PR.

File: crates/relay/src/benchmarking.rs (hunk 1 above)
Comment (gregdhill) [reply crates/relay/src/benchmarking.rs:28]: Yep the intent of this PR is to just get the benchmarks to run again, but I used the issue benchmarks to accomplish this.

**Review by sander2 — APPROVED** [review]

**Pull request merged**